bin/test_client.rb

```diff
@@ -106,49 +108,54 @@ def field_should_be_nil(fields, obj, method_name)
 
 def expected_fields_in_email_resp
   %w(id
-     api_key
-     billable_units
-     to
-     subject
-     body
-     notification_type
+     reference
+     email_address
+     type
      status
-     service
      sent_at
-     sent_by
      template
```

Comment:
we should drill down into template and assert that the id, version and uri are set